pages/index.tsx

```diff
@@ -60,16 +69,21 @@ export default function Home({ onLike, likes }: HomeProps): JSX.Element {
 
       {/* Switching between the three different views */}
 
-      {viewPoint.random && <RandomView />}
+      {viewPoint.random && <RandomView artists={artists} />}
       {viewPoint.artists &&
         artists.map((artist) => (
-          <List key={artist.id} {...artist} onLike={onLike} likes={likes} />
+          <List key={artist._id} {...artist} onLike={onLike} likes={likes} />
```

Comment:
This could be slightly optimised:
Instead of passing the whole `likes` array "[23, 221, 12, 3, 2]" into each entry, you could make the assertion, if a artist is liked, already here one level up:
```suggestion
          <List key={artist._id} {...artist} onLike={onLike} isLiked={likes.includes(artist._id)} />
```

Reply:
Great idea, much cleaner!